app/js/App.js

```diff
@@ -20,11 +20,11 @@ class App {
    * Start application by creating a page
    */
 
-  constructor() {
+  constructor(theme = 'default') {
```

Comment:
I think `App`'s constructor should take an instance of `Settings`. The default theme should be handled in the `Settings` class.